deploy-board/deploy_board/webapp/host_views.py

```diff
@@ -63,7 +63,7 @@ def get_asg_name(request, hosts):
 
 def get_show_terminate(hosts):
     for host in hosts:
-        if host and host.get('state') and host.get('state') != 'PENDING_TERMINATE' and host.get('state') != 'TERMINATING' and host.get('state') != 'TERMINATED':
+        if host and host.get('state') and host.get('state') not in ('PENDING_TERMINATE', 'TERMINATING', 'TERMINATED', 'PENDING_TERMINATE_NO_REPLACE'):
```

Comment:
Could use a set instead of tuple

```suggestion
        if host and host.get('state') and host.get('state') not in {'PENDING_TERMINATE', 'TERMINATING', 'TERMINATED', 'PENDING_TERMINATE_NO_REPLACE'}:
```